src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -130,7 +140,7 @@ private Path resolvePath(Path path, String errorMessage) throws IOException {
      * @return true if the processed ended succesfully, false otherwise
      * @throws IOException in case default path for Substrate dependencies can't be created
      */
-    private boolean setupDependencies() throws IOException {
+    public boolean setupDependencies() throws IOException {
```

Comment:
I'm not sure it we should expose this method... We access it through `getJavaSDKLibsPath()` and `getJavaFXSDKLibsPath()`, as we also need to get the path to those libraries, and when not found, we download them with `setupDependencies()`.

In case of Android, I'd do something similar:

```

public Path getAndroidSDKPath() throws IOException {
        return resolvePath(configuration.getAndroidSdkPath(),"Fatal error, could not install Android SDK ");
    }
public Path getAndroidNDKPath() throws IOException {
        return resolvePath(configuration.getAndroidNdkPath(),"Fatal error, could not install Android NDK ");
    }
```